ReleaseNotes-2.10

```diff
@@ -28,107 +28,28 @@ Features
-   - FISSILE build format
-   - AppImage build format
- * freezelink command to freeze current sources accessed via project link
- * support showing source files in blame view (works also via links)
- * support project copy with makeoriginolder option
- * support automatic vrev extending via project links
- * Improved container support:
-   - support build of layered containers by reusing existing contaienrs
-   - support publishing to docker registry server
-   - support container signing via notary server
- * cloud upload server supporting Amazon EC2 and Microsoft Azure
- * improved bootstrap cycle handling
- * additional SHA256 checksum in source commit handling for security
- * projects can be temporary suspended to avoid scheduling between multiple changes
- * support AirBrake for reporting problems
- * support new debian repository format
- * support for building in openstack cloud
- * Many smaller improvements in DownloadOnDemand and multibuild handling
 
 Shipment:
- * To make use of the ec2 cloud upload feature you need to:
-   - Enable the Public Cloud module (only needed for SLE12 systems).
-   - Install the obs-cloud-uploader package.
 
 Bugfixes:
- * Fix deletion of groups with users.
- * Fix notification generation with very big payloads.
- * Create history element on priority raise of request.
- * Fix huge bottleneck in notification emails.
- * Fix setting of new attributes to a project or package.
 
 Intentional changes:
 ====================
 
- * Creating of repositories on branching has changed if repositories of the source refer each other. This gets recreated in new project.
-
- * Project copy is not adding the user anymore.
-
- * Service dispatcher is used by default now.
-
- * The editing of a user's realname, email adress or password is no longer possible if LDAP mode is activated.
-
- * Unused ldap options in options.yml were dropped:
-   - ldap_update_support
-   - ldap_object_class
-   - ldap_entry_base
-   - ldap_sn_attr_required
-
- * Extended build number scheme is enforced for maintenance updates via new project link
-   attribute extendvrev now
-
- * Dropping of the project/package tag functionality/api
-
- * Password hashing algorithm was changed to bcrypt (blowfish).
-
- * The backend notification plugin system is not used anymore:
-   - The RabbitMQ plugin is replaced with a RabbitMQ message bus implementation in the frontend, you can find details about this in the admin manual.
-   - The Hermes plugin is dropped without replacement as it was only used for notifications which the OBS is doing on it's own since quite some time.
-
- * _multibuild files have a changed syntax, but OBS stays backward compatible.  Use <flavor> instead of <package> elements in the future.
-
- * publish hook failures are handled as fatal failures now.
-   => publisher will retry to publish
+ * 
 
 Other changes
 =============
 
- *  Binaries page now points to the download, cloud upload and details page using links.
+ * 
```

Comment:
why not removing this sections instead? :thinking:

Reply:
Whe you say "those sections" what sections are you referring to?